### PR TITLE
Extend Unit Preasure

### PR DESCRIPTION
**Pull request by schulz3000**
Add additional prefixes to Pascal and Bar

### CODE REVIEW

File: UnitsNet/Scripts/UnitDefinitions/Pressure.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
             "PluralName": "Pascals",
             "FromUnitToBaseFunc": "x",
             "FromBaseToUnitFunc": "x",
-            "Prefixes": ["Kilo","Mega"],
+            "Prefixes": ["Micro","Deca","Hecto","Kilo","Mega","Giga"],
```
Comment (anjdreas) [inline UnitsNet/Scripts/UnitDefinitions/Pressure.json]: Any particular reason why Milli and Centi is left out? I don't know what prefixes are commonly used with Pascal.

File: UnitsNet/Scripts/UnitDefinitions/Pressure.json [hunk 2]
```diff
@@ -42,6 +42,7 @@
             "PluralName": "Bars",
             "FromUnitToBaseFunc": "x*1e5",
             "FromBaseToUnitFunc": "x/1e5",
+            "Prefixes": ["Milli","Centi","Deci","Kilo","Mega"],
```
Comment (anjdreas) [inline UnitsNet/Scripts/UnitDefinitions/Pressure.json]: Same here, did you intentionally leave Hecto out? Not commonly used?

File: UnitsNet/Scripts/UnitDefinitions/Pressure.json (hunk 1 above)
Comment (schulz3000) [inline UnitsNet/Scripts/UnitDefinitions/Pressure.json]: Of course it's possible to convert it to Milli and Centi but I never see it in the wild. In my company we we are working lot with measuring of preasure. There I also never see a device that use MilliPascal or CentiPascal.
As source in the internet I only find a german wiki entry where you can also find these prefixes: http://de.wikipedia.org/wiki/Pascal_(Einheit)

File: UnitsNet/Scripts/UnitDefinitions/Pressure.json (hunk 2 above)
Comment (schulz3000) [inline UnitsNet/Scripts/UnitDefinitions/Pressure.json]: Same as for Pascal
Possible source: http://en.wikipedia.org/wiki/Bar_(unit)

**Comment by anjdreas** [comment]
Ok, good enough for me.

You need to fix that compile error in the test project though. Basically implement the test cases in PressureTests.cs. The wiki on adding new units explains this for your reference.

Build log:
http://dev.swingcatalyst.com/TeamCity/viewLog.html?buildId=22246&buildTypeId=UnitsNet

Let me when it's fixed and I'm ready to merge this in.

**Comment by schulz3000** [comment]
@anjdreas Unittests are fixed

**Comment by anjdreas** [comment]
Looks good, thanks!

**Pull request merged**